source/installation-guide/more-installation-alternatives/splunk/splunk-app.rst

```diff
@@ -66,7 +66,19 @@ Installation
       :align: center
       :width: 100%
 
-When the Wazuh app for Splunk is installed, the next step consists on installing and configuring :ref:`Splunk forwarder <splunk_forwarder>`.
+    When the Wazuh app for Splunk is installed, the next step consists on installing and configuring :ref:`Splunk forwarder <splunk_forwarder>`.
+
+#. In order to configure the index and source type of the app, we must go to *Settings/Index* (view image).
+
+    The default values are for index ``wazuh`` and for source type ``*``.
+
+    .. note::
+      The Wazuh app considers as valid index all indexes that contain the word ``wazuh`` in the source type, the default value for source type is ``wazuh``.
```

Comment:
I think it could be better with 
` The Wazuh app considers as valid index all indexes that contain the word ``wazuh`` in the source type, the default value from the installation guide for source type is ``wazuh``

Reply:
Thanks for the observation now I modify it.